internal/command/flag.go

```diff
@@ -105,7 +112,12 @@ func newCommonFlags() (*CommonFlags, error) {
 	flagSet.StringVar(&flags.AppVersion, "app-version", appVersion, "Version number in the form x, x.y or x.y.z semantic version")
 	flagSet.StringVar(&flags.CacheDir, "cache", cacheDir, "Directory used to share/cache sources and dependencies")
 	flagSet.BoolVar(&flags.NoCache, "no-cache", false, "Do not use the go build cache")
-	flagSet.Var(&flags.Engine, "engine", "The container engine to use. Supported engines: [docker, podman]. Default to autodetect.")
+	flagSet.BoolVar(&flags.NoProjectUpload, "no-project-upload", false, "Will reuse the project data available in S3")
```

Comment:
I wonder if these parameters should be more clearly related only to k8s?

Reply:
Also I think they should be behind the "k8s" flag to avoid confusion.

Reply:
> Also I think they should be behind the "k8s" flag to avoid confusion.

Oh, indeed, didn't think of that, but it makes sense.

Reply:
Not to sure about how to make them more clearly related to k8s. Maybe something for later when we do the migration to cli.Arg.

Reply:
Mentioning it in the docs would be a start.
Possibly putting it in the parameter name might be possible too, if it does not make them too long

Reply:
Tried in the parameter name and it feels weird. I have updated the documentation to reflect it is linked to the kubernetes engine for each parameters.